.github/workflows/combine-typos.yml

```diff
@@ -0,0 +1,48 @@
+name: Combine Typo Pull Requests
+
+on:
+  pull_request:
+    types: [ labeled ]
```

Comment:
Maybe having that on cron/weekly would be great ?

Reply:
Bi weekly might be better

Reply:
The idea is that this PR "hoovers" typofix PRs and _instantly_ adds it to a batch PR (which can then be reviewed/merged weekly, or whenever it's substantial enough).